specs/latest/2.0/index.html

```diff
@@ -3386,6 +3492,19 @@ <h2 class="no-toc">Table of contents</h2>
         but that can be achieved by adjusting the data store offset accordingly.
     </div>
 
+    <h3><a name="DOM_UPLOAD_UNPACK_PARAMS">Pixel store parameters for <code>TexImageSource</code> uploads</a></h3>
+    <p>
+        <code>UNPACK_ALIGNMENT</code> and <code>UNPACK_ROW_LENGTH</code> are
```

Comment:
I agree with @zhenyao that it would be easier to explain if these entry points took sourceOffsetX and sourceOffsetY, and ignored all of the pixel unpack parameters.